src/specter_edn/core.clj

```diff
@@ -7,7 +7,9 @@
              [graph :as g]]
             [rewrite-clj
              [node :as n]
-             [parser :as p]]))
+             [parser :as p]]
+            [rewrite-clj.node.comment :as c]
```

Comment:
I think these functions are re-exported from `rewrite-clj.node`, so you don't need to import `node.comment` or `node.whitespace`.